cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -88,23 +106,24 @@ func (g *GetEventsRequest) Valid(maxLimit uint) error {
 	// Validate start
 	// Validate the paging limit (if it exists)
 	if g.Pagination != nil && g.Pagination.Cursor != nil {
-		if g.StartLedger != 0 {
-			return errors.New("startLedger and cursor cannot both be set")
+		if g.StartLedger != 0 || g.EndLedger != 0 {
+			return errors.New("startLedger/endLedger and cursor cannot both be set") //nolint:forbidigo
 		}
 	} else if g.StartLedger <= 0 {
 		return errors.New("startLedger must be positive")
 	}
+
 	if g.Pagination != nil && g.Pagination.Limit > maxLimit {
 		return fmt.Errorf("limit must not exceed %d", maxLimit)
 	}
 
 	// Validate filters
-	if len(g.Filters) > 5 {
+	if len(g.Filters) > maxFiltersLimit {
 		return errors.New("maximum 5 filters per request")
 	}
 	for i, filter := range g.Filters {
 		if err := filter.Valid(); err != nil {
-			return errors.Wrapf(err, "filter %d invalid", i+1)
+			return errors.Wrapf(err, "filter %d invalid", i+1) //nolint:forbidigo
```

Comment:
use https://github.com/pkg/errors instead of our support/errors and the linter will :shushing_face: